webrtc-c/canary/src/CanaryWebrtc.cpp

```diff
@@ -11,15 +11,41 @@ VOID handleSignal(INT32 signal)
     terminated = TRUE;
 }
 
+// add frame pts, original frame size, CRC to beginning of buffer after Annex-B format NALu
```

Comment:
Care must be taken to avoid zeroes. Recommend encoding the metadata into base64 to avoid NALu escaping

Reply:
Makes sense! Fixed.